### DISCUSSION TITLE
Setup Date format globally by locale

### DISCUSSION THREAD

**Discussion by ghena** [bagisto/bagisto#6711] [Ideas]
Hi guys,

there is a way to change all format data displayed in frontend - backend linked to APP_LOCALE or language ?

For example: field created_at should be rendered dynamically based on APP_LOCALE or language.

Please let me know.
Regards.
.g

**Comment by devansh-webkul** [comment]
Hi @ghena,

Currently, we have no plan for this, you can customize it as per your need. We will surely implement this.

Right now I just moved this to the discussion.

**Comment by ghena** [comment]
Hi this features has been released ?
